e2e/tests-dfx/assetscanister.bash

```diff
@@ -539,3 +539,19 @@ CHERRIES" "$stdout"
     assert_not_match "etag: my-etag"
     assert_match "etag: \"[a-z0-9]{64}\""
 }
+
+@test "asset configuration via .ic-assets.json5 - detect unused config" {
+    install_asset assetscanister
+
+    dfx_start
+
+    echo '[
+      {
+        "match": "nevermatchme",
+        "cache": { "max_age": 2000 }
+      }
+    ]' > src/e2e_project_frontend/assets/.ic-assets.json5
+
+    dfx deploy
+    assert_contains "Unused"
```

Comment:
Somehow this doesn't work, not sure why. Output from local run:
```
Creating canisters...
   Creating canister e2e_project_backend...
.....................................
   WARN: Unused config: AssetConfigRule { r#match: /private/var/folders/qr/2sz_4fyx75z7smnl9k554p000000gn/T/dfx-e2e-XXXXXXXX.anTWMxGj/working-dir/e2e_project/src/e2e_project_frontend/assets/nevermatchme, cache: Some(CacheConfig { max_age: Some(2000) }), headers: Absent, ignore: None, used: false, origin: "/private/var/folders/qr/2sz_4fyx75z7smnl9k554p000000gn/T/dfx-e2e-XXXXXXXX.anTWMxGj/working-dir/e2e_project/src/e2e_project_frontend/assets" }
   Starting batch.
   Staging contents of new and changed assets:
     /sample-asset.txt 1/1 (24 bytes) sha 2d523f5aaeb195da24dcff49b0d560a3d61b8af859cee78f4cff0428963929e6
     /binary/noise.txt 1/1 (19 bytes) sha c7129a1e7a6d66c69670421e73655007caf1673febdcbe3eefec86653a4e5e44
     /text-with-newlines.txt 1/1 (36 bytes) sha f3bf72b15312907983266db759f4788835bb0e4a087056647308b39b454e5fa0
   Committing batch.
   Deployed canisters.
   URLs:
     Backend canister via Candid interface:
       e2e_project_backend: http://127.0.0.1:61689/?canisterId=r7inp-6aaaa-aaaaa-aaabq-cai&id=rrkah-fqaaa-aaaaa-aaaaq-cai

   -- output does not match --
   search phrase : Unused
   actual output :
   --
```

Reply:
I also tried `assert_match`, and also tried 
```
dfx deploy | tee deploy.log
cat deploy.log
assert_....
``` 
to make sure its not the problem with stdout/stderr

Reply:
please help